tests/test_xyz/test_points_from_wells.py

```diff
@@ -112,14 +131,21 @@ def test_get_zone_thickness_some_wells(testpath, tmp_path):
         filter={"ZoneName": ["SO622"]},
     )
 
+    # not the order in the dataframe may vary randomly so do a sort
+    dfr = mypoints.dataframe.sort_values(["WellName", "Zone", "Z_TVDSS"])
+    dfr.reset_index(inplace=True)
+    logger.info(dfr)
+
+    assert dfr["X_UTME"][0] == pytest.approx(462698.2375)
+    assert dfr["WellName"][0] == "OP_1"
+    assert dfr["X_UTME"][20] == pytest.approx(462749.4820)
+    assert dfr["WellName"][20] == "OP_5"
```

Comment:
This already appears to be a snapshot test, so perhaps go all the way to be explicit?

Reply:
Done